src/emulator/opcodes_table.cpp

```diff
@@ -17,6 +17,13 @@ OpCodesTable::OpCodesTable()
         opcodes_[i] = &OpCodesTable::OpNotImplemented<&OpCodesTable::AddressingModeNone>;
 
     opcodes_[0x00] = &OpCodesTable::OpBRK;
+    opcodes_[0x08] = &OpCodesTable::OpPHP;
```

Comment:
I wonder if it's not better to include the template for each of these, and specifically identify and run the implied addressing mode.  I think it'll help make the code more readable and self-documenting.